"""fix the foreign  key mistake

Revision ID: f73289069641
Revises: 1374784e74d0
Create Date: 2025-04-05 20:40:15.022093

"""
from alembic import op
import sqlalchemy as sa


# revision identifiers, used by Alembic.
revision = 'f73289069641'
down_revision = '1374784e74d0'
branch_labels = None
depends_on = None


def upgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    with op.batch_alter_table('school_data', schema=None) as batch_op:
        batch_op.drop_constraint('school_data_num_days_fkey', type_='foreignkey')

    # ### end Alembic commands ###


def downgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    with op.batch_alter_table('school_data', schema=None) as batch_op:
        batch_op.create_foreign_key('school_data_num_days_fkey', 'classes', ['num_days'], ['id'])

    # ### end Alembic commands ###
